composer.json

```diff
@@ -30,17 +30,13 @@
         "php": "^8.1",
         "ext-mbstring": "*",
         "guzzlehttp/psr7": "^2.4",
-        "jetbrains/phpstorm-attributes": "^1.0",
         "psr/container": "^2.0",
         "psr/event-dispatcher": "^1.0",
         "psr/http-client": "^1.0",
         "psr/http-message": "^1.0|^2.0",
         "psr/log": "^1.0|^2.0|^3.0",
         "symfony/console": "^5.4|^6.0",
-        "symfony/var-dumper": "^5.4|^6.4",
```

Comment:
Isn't it in use?

Reply:
I don't found. And composer require checker is passed.